tests/sentry/incidents/action_handlers/test_opsgenie.py

```diff
@@ -33,13 +34,25 @@ def setUp(self):
         self.org_integration.config = {"team_table": [self.og_team]}
         self.org_integration.save()
 
+        resp_data = {
+            "result": "Integration [sentry] is valid",
+            "took": 0.093,
```

Comment:
are these numbers and ID's meaningful in any way? if not, could we make it obvious they're fillers (like use 'foo' or something) or move them into separate variables?